.code-samples.meilisearch.yaml

```
@@ -471,10 +452,8 @@ faceted_search_walkthrough_attributes_for_faceting_1: |-
         ]
     }'
 faceted_search_walkthrough_facet_filters_1: |-
-  $ curl --get 'http://localhost:7700/indexes/movies/search' \
-      --data-urlencode 'q=thriller' \
-      --data-urlencode 'facetFilters=[["genres:Horror", "genres:Mystery"], "director:Jordan Peele"]'
+  $ curl 'http://localhost:7700/indexes/movies/search' \
+  --data '{ "q": "thriller", "facetFilters": [[\"genres:Horror\", \"genres:Mystery\"], \"director:Jordan Peele\"] }'
 faceted_search_walkthrough_facets_distribution_1: |-
-  $ curl --get 'http://localhost:7700/indexes/movies/search' \
-      --data-urlencode 'q=Batman' \
-      --data-urlencode 'facetsDistribution=["genres"]'
+  $ curl 'http://localhost:7700/indexes/movies/search' \
+  --data '{  "q": "Batman", "facetsDistribution": [\"genres\"] }'
```

```suggestion
  --data '{  "q": "Batman", "facetsDistribution": ["genres"] }'
```

`{"message":"Invalid JSON: expected value at line 1 column 42","errorCode":"bad_request","errorType":"invalid_request_error","errorLink":"https://docs.meilisearch.com/errors#bad_request"}%`